sdk/python/feast/constants.py

```diff
@@ -129,10 +131,22 @@ class AuthProvider(Enum):
     # Authentication Provider - Google OpenID/OAuth
     CONFIG_AUTH_PROVIDER: "google",
     CONFIG_SPARK_LAUNCHER: "dataproc",
-    CONFIG_SPARK_INGESTION_JOB_JAR: "gs://feast-jobs/spark/ingestion/feast-ingestion-spark-develop.jar",
+    CONFIG_SPARK_INGESTION_JOB_JAR: "https://storage.googleapis.com/feast-jobs/spark/"
+    "ingestion/feast-ingestion-spark-develop.jar",
     CONFIG_SPARK_STANDALONE_MASTER: "local[*]",
     CONFIG_REDIS_HOST: "localhost",
     CONFIG_REDIS_PORT: "6379",
     CONFIG_REDIS_SSL: "False",
     CONFIG_SPARK_HISTORICAL_FEATURE_OUTPUT_FORMAT: "parquet",
+    # Enable or disable TLS/SSL to Feast Service
+    CONFIG_JOB_SERVICE_ENABLE_SSL_KEY: "False",
+    # Path to certificate(s) to secure connection to Feast Job Service
+    CONFIG_JOB_SERVICE_SERVER_SSL_CERT_KEY: "",
+    # IngestionJob StatD Config
+    CONFIG_STATSD_HOST: "",
+    CONFIG_STATSD_PORT: "0",
+    # IngestionJob DeadLetter Destination
+    CONFIG_DEADLETTER_PATH: "",
+    # ProtoRegistry Address
```

Comment:
I think we need to have at least some documentation/information on what Stencil is. At least a link to the repo.